src/eduid_webapp/personal_data/schemas.py

```diff
@@ -43,9 +44,9 @@
 
 class PersonalDataRequestSchema(EduidSchema, CSRFRequestMixin):
 
-    given_name = fields.String(required=True)
-    surname = fields.String(required=True)
-    display_name = fields.String(required=True)
+    given_name = fields.String(required=True, validate=validate_required)
```

Comment:
Isn't given_name, surname and display_name already required? Is this just to get the error message "pdata.field_required"?

Reply:
They are already required, but can be empty. So the validator should probably be called validate_nonempty, I'll change the name. However, the message to the user must still be that the field is required...